app/wasm_filter.go

```diff
@@ -0,0 +1,35 @@
+package app
+
+import (
+	"strings"
+
+	"github.com/Finschia/finschia-sdk/codec"
+	codectypes "github.com/Finschia/finschia-sdk/codec/types"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	sdkerrors "github.com/Finschia/finschia-sdk/types/errors"
+	wasmkeeper "github.com/Finschia/wasmd/x/wasm/keeper"
+	wasmtypes "github.com/Finschia/wasmd/x/wasm/types"
+	wasmvmtypes "github.com/Finschia/wasmvm/types"
+)
+
+// deniedModulesInStargateMsg is deny list in StargateMsg of wasm
+var deniedModulesInStargateMsg = []string{"/lbm.fswap.v1", "/lbm.fbridge.v1"}
```

Comment:
Apparently, this var is only used in filteredStargateMsgEncoders. Why don't you put it in there?

Reply:
Exactly, this var is used in `wasmFilteredEncodeStargateMsg`. I defined it as a global variable considering further modifications later. But since it only used within `wasmFilteredEncodeStargateMsg`, it seems appropriate now to move it into the `wasmFilteredEncodeStargateMsg` function.
Thank you.